app/components/Featured.tsx

```diff
@@ -61,8 +64,49 @@ const Featured = ({ newestFeaturedContent }: FeaturedProps) => {
                   </div>
                 )}
               </div>
-            );
-          })}
+            ))}
+          </div>
+
+          <div className="grid grid-cols-1 gap-7 md:grid-cols-2">
+            {verticalFeaturedContent?.map((content) => (
+              <div key={content.id} className="my-5">
+                <Card cssClass="bg-variant-blue-4 sm:h-[32rem]">
+                  <div className="grid h-full">
```

Comment:
Tror kanskje det bør være noe styling på bredde. Kan nok med fordel ha 2 kolonner på brede skjermer (spesielt over 3000px, men kanskje også før)

![bilde](https://user-images.githubusercontent.com/10518767/236793958-d27553ee-6c65-4285-8896-55752cfa3121.png)

Reply:
Ja, men er det noe som kanskje bør ligge på dashboard istedenfor her, slik at det påføres flere deler av applikasjonen?

Reply:
Om jeg har forstått kodestrukturen rett, ville jeg tenkt det er mest logisk å legge det inn i `Featured`, siden det handler om de spesifikke kortene, og de andre kortene er i egne parents?

Reply:
dashboard er parent route til både Featured og de ulike kategoriene, alle disse rendres i Outlet komponenten til dashboard. Jeg foreslår at vi heller legger til en maks bredde på innholdet istedenfor å ha to kort i bredden, fordi jeg er litt usikker på hvordan det blir med to liggende kort i bredden 😅

Reply:
Den er jeg enig i, at vi heller bør sette en maksbredde.

Reply:
Kan ta det på responsivt-design-issues, så godkjenner denne nå